torch/distributed/_composable/fsdp/_fsdp_common.py

```diff
@@ -150,3 +151,16 @@ def _cast_fp_tensor(dtype: torch.dtype, x: torch.Tensor) -> torch.Tensor:
     ):
         return x
     return x.to(dtype)
+
+
+def _get_device_handle_from_device(device: Optional[torch.device] = None):
+    if device is None:
+        device_type = "cuda" if torch.cuda.is_available() else "cpu"
+    else:
+        device_type = device.type
+
+    device_handle = _get_device_handle(device_type)
+
+    if device_handle is None:
+        raise RuntimeError("InValid device handle for device type:", device)
```

Comment:
if you decide to keep this
```suggestion
        raise RuntimeError("Invalid device handle for device type:", device)
```

Reply:
will fix this